lib/filters/sanitize-field-names.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+const querystring = require('querystring')
+
+const HEADER_FORM_URLENCODED = 'application/x-www-form-urlencoded'
+
+/**
+ * Handles req.body as object, string, or Buffer
+ *
+ * Express provides multiple body parser middlewares with x-www-form-urlencoded
+ * handling.  See http://expressjs.com/en/resources/middleware/body-parser.html
+ */
+function removeKeysFromPostedFormVariables (body, requestHeaders, regexes) {
```

Comment:
This function is probably more complicated than it needs to be right now.  I originally thought we would need to be able to handle the parsed request body in all its possible formats. The way feature development ending up going the agent had already normalized these values by the time we wanted to handle them.  This could probably be simplified to remove the raw/buffer handling, but I decided to leave this code in place as it may be useful when/if we handle the request body parsing for fastify, hapi, and koa.   I was 52%/48% on this -- not a strongly held decision.